### DISCUSSION TITLE
Prompt first-time users to run `aura check`

### DISCUSSION THREAD

**Discussion by abdeoliveira** [fosskers/aura#892] [Ideas]
As per title.

I notice some issues could be easily spotted if users would follow the manual and run aura check just after installation. Thus my suggestion is to identify those who are running aura just after install and :

1. Display a message to the the user that "it seems this is the first time you are running aura" or something.
2. Stating that check will be performed in the system.
3. Prompt the user for [Y/n].

**Comment by fosskers** [comment]
A decent suggestion. One issue would be that forcing a prompt upon first interaction with Aura 4 would break existing scripting workflows. I want the migration from 3 to 4 to be seamless.

**Reply by abdeoliveira** [reply]
It would be possible to identify a 3.0 install at first-run on 4.0?

**Reply by fosskers** [reply]
I think the implementation of that would be quite complex. At the moment I think it's fine to expect the users to read the documentation. If they're not willing to read the [post-installation instructions](https://github.com/fosskers/aura?tab=readme-ov-file#post-installation), then it's mea culpa for them.

**Comment by abdeoliveira** [comment]
Fair enough :-)
